PyTorch/Segmentation/MaskRCNN/pytorch/maskrcnn_benchmark/engine/trainer.py

```diff
@@ -146,8 +146,9 @@ def do_train(
 
     total_training_time = time.time() - start_training_time
     total_time_str = str(datetime.timedelta(seconds=total_training_time))
+    sec_per_iteration = total_training_time / max_iter
     logger.info(
-        "Total training time: {} ({:.4f} s / it)".format(
-            total_time_str, total_training_time / (max_iter)
+        "Total training time: {} ({:.4f} s / it) throughput: {:.2f} FPS".format(
+            total_time_str, sec_per_iteration, int(cfg.SOLVER.IMS_PER_BATCH) / sec_per_iteration
```

Comment:
maybe you can add a new variable sample_per_sec = int(cfg.SOLVER.IMS_PER_BATCH) / sec_per_iteration, then pass that in, which is more clear

Reply:
Done